### PR TITLE
Stylish haskell applied

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: cardano-cli/src/Cardano/CLI/Byron/Commands.hs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE GeneralisedNewtypeDeriving #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
```
Comment (erikd) [inline cardano-cli/src/Cardano/CLI/Byron/Commands.hs:1]: Is this commit still needed?

File: cardano-cli/test/Test/OptParse.hs [hunk 2]
```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE FlexibleContexts #-}
```
Comment (erikd) [inline cardano-cli/test/Test/OptParse.hs:1]: What change made this needed?

File: .stylish-haskell.yaml [hunk 3]
```diff
@@ -15,41 +15,13 @@ steps:
   #     # true.
   #     add_language_pragma: true
 
-  # Format record definitions. This is disabled by default.
```
Comment (erikd) [inline .stylish-haskell.yaml]: This should probably be squashed down into one of the previous commits.

File: cardano-cli/src/Cardano/CLI/Byron/Commands.hs (hunk 1 above)
Comment (newhoggy) [reply cardano-cli/src/Cardano/CLI/Byron/Commands.hs]: I've removed the pragma changes to see what happens.

File: cardano-cli/test/Test/OptParse.hs (hunk 2 above)
Comment (newhoggy) [reply cardano-cli/test/Test/OptParse.hs]: I've removed the pragma changes to see what happens in CI.

File: .stylish-haskell.yaml (hunk 3 above)
Comment (newhoggy) [reply .stylish-haskell.yaml]: Squashed!

File: cardano-cli/test/Test/OptParse.hs (hunk 2 above)
Comment (newhoggy) [reply cardano-cli/test/Test/OptParse.hs]: Looks to be working.

File: cardano-cli/src/Cardano/CLI/Byron/Commands.hs (hunk 1 above)
Comment (newhoggy) [reply cardano-cli/src/Cardano/CLI/Byron/Commands.hs]: Looks to be working.

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**